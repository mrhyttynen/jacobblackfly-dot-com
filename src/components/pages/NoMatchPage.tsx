export const NoMatchPage = () => {
  return <p>There's nothing here: 404!</p>;
};
