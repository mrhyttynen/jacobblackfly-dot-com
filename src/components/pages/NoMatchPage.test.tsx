import React from "react";
import { render, screen } from "@testing-library/react";
import { NoMatchPage } from "./NoMatchPage";
import renderer from "react-test-renderer";

describe("NoMatchPage", () => {
  it("renders page that matches snapshot", () => {
    const tree = renderer.create(<NoMatchPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
