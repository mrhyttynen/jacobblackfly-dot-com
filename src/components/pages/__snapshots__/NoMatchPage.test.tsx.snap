// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatchPage renders page that matches snapshot 1`] = `
<p>
  There's nothing here: 404!
</p>
`;
